src/main/java/org/opentripplanner/standalone/CommandLineParameters.java

```diff
@@ -61,6 +61,13 @@
             description = "Pass the graph to the server in-memory after building it, without saving to disk.")
     public boolean inMemory;
 
+    @Parameter(names = { "--skipLinking" }, description = "Skip the streetLinkerModule. When building graphs in " +
+            "several steps, you only want to do the linking once.")
+    public boolean skipLinking = false;
+
+    @Parameter(names = { "--graphName" }, description = "Specify the file name the graph should be saved as.")
+    public String graphName = "Graph.obj";
+
```

Comment:
I'm not seeing anywhere where `graphName` is used?

Reply:
Should be fixed now